bundles/org.openhab.binding.mybmw/src/main/java/org/openhab/binding/mybmw/internal/handler/backend/JsonStringDeserializer.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.mybmw.internal.handler.backend;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.mybmw.internal.dto.charge.ChargingSessionsContainer;
+import org.openhab.binding.mybmw.internal.dto.charge.ChargingStatisticsContainer;
+import org.openhab.binding.mybmw.internal.dto.remote.ExecutionStatusContainer;
+import org.openhab.binding.mybmw.internal.dto.vehicle.VehicleBase;
+import org.openhab.binding.mybmw.internal.dto.vehicle.VehicleStateContainer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+/**
+ * 
+ * deserialization of a JSON string to a Java Object
+ * 
+ * @author Martin Grassl - initial contribution
+ */
+@NonNullByDefault
+public interface JsonStringDeserializer {
+
+    static final Logger LOGGER = LoggerFactory.getLogger(JsonStringDeserializer.class);
+
+    static final Gson GSON = new Gson();
+
+    public static List<VehicleBase> getVehicleBaseList(String vehicleBaseJson) {
+        try {
+            VehicleBase[] vehicleBaseArray = deserializeString(vehicleBaseJson, VehicleBase[].class);
+            return Arrays.asList(vehicleBaseArray);
+        } catch (JsonSyntaxException e) {
+            LOGGER.warn("JsonSyntaxException {}", e.getMessage());
```

Comment:
to be honest I would like to keep it. It would blow up the code of the myBMWHttpProxy as anyhow I have to do a try/catch for the exception. In the proxy class it looks much more cleaner if I just have one method call instead of having the try/catches. In this case the caller doesn't have to care about the exception as an empty object will be returned in case of an error. 
But I can do it for sure if you prefer it.

Reply:
OK. But I guess the user won't have to see this warning with "JsonSyntaxException" which seems quite technical and alarming, so maybe it should be reduced to log level **debug**?

Reply:
OK, done